webapp/components/Comment/Comment.vue

```diff
@@ -86,6 +96,7 @@ export default {
     ContentMenu,
     ContentViewer,
     CommentForm,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T10:28:58Z" title="Thursday, January 16th 2020, 11:28:58 am +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -141,6 +148,10 @@ export default {
     },
   },
   methods: {
+    reply(comment) {
```


```suggestion
    directReplyToComment(comment) {
```

I would agree with @Tirokk on this and favor verbosity here because there could be many different meanings of reply in the future

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-16T11:15:13Z" title="Thursday, January 16th 2020, 12:15:13 pm +01:00">Jan 16, 2020</time>_

---


@mattwr18 maybe we want to reply to the author of the post some day? I'm leaning towards expressing the meaning through the name of the argument, ie. `reply(comment)` or `reply(author)`. :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-31T11:52:35Z" title="Friday, January 31st 2020, 12:52:35 pm +01:00">Jan 31, 2020</time>_

---


ok, I'm fine with that, I know not everyone favors verbosity as well, I certainly wouldn't request changes for something like this.